documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -0,0 +1,37 @@
+# Opam Support
+
+## Related properties
+
+[Detector properties](../properties/detectors/opam.md)
+
+[detect_product_short] has two detectors for Opam:
+
+* OPAM CLI Detector
+* OPAM Lock Detector
+
+## OPAM CLI Detector
+
+* This detectable executes opam commands to discover dependencies of opam projects.
+
+* This OPAM detector will be executed on your project if [detect_product_short] finds `.opam` file in your top level directory. It requires `opam`
+exe to be present on your $PATH. You can also override the location for `opam` exe by the OPAM path property.
+
+The OPAM Build Detector will work in the following way on your project:
+
+1. [detect_product_short] OPAM Build Detector will run `opam --version` to get version of opam on your machine.
+2. If the version of opam is greater than or equal to 2.2.0, then [detect_product_short] would run `opam tree . --with-test --with-doc --with-dev --recursive`
+to get list of resolved packages installed in the current switch for the project.
```

Comment:
... to get **the** list ...
Could we turn the word "switch" into a link that directs users to the opam docs describing a switch? 
i.e. [switch](https://ocaml.org/docs/opam-switch-introduction#opam-switch-introduction)